packages/accordions/stories/examples/Uncontrolled.tsx

```diff
@@ -109,6 +109,12 @@ Uncontrolled.args = {
   isBare: false
 };
 
+Uncontrolled.argTypes = {
+  defaultExpandedSections: {
+    control: { disable: true }
```

Comment:
Can we open this control up to allow the demo to be affected by it?

Reply:
Unfortunately, the demo would not be affected by the knob because the knob value used as initial state for a `useState` in `Accordion`. The result is that the demo would render correctly for the initial `defaultExpandedSections`, but would not show updates for subsequently different `defaultExpandedSections`. I think adding a `defaultValue` knob to the `Input` would have the same issues.

I think if the demo forced a re-render of the `Accordion` each time the `defaultExpandedSections` changed, then the demo would reflect the knob. But this requires some one-off code to live in the story.

Reply:
got it.